pyfr/integrators/dual/pseudo/densepseudoschemes/__init__.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from pkg_resources import resource_listdir, resource_string
+import re
+
+
+def _get_coefficients(scheme):
+        a = []
+
+        scheme = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', scheme)
+        scheme = re.sub(r'\(|\)|,', '', scheme).strip()
+
+        for l in scheme.splitlines():
+            if not l:
+                continue
+
+            a.append([float(f) for f in l.split()])
+
+        # The last row is the b vector
+        return a[:-1], a[-1]
+
+def get_densepseudoscheme(namein, porderin):
+    rpaths = resource_listdir(__name__, 'schemes')
```

Comment:
This should not be here.  Once the classes are registered (created with type) the rest of the code should not need to make a distinction between different types of pseudo schemes.